### PR TITLE
Add 2.071.2.s1 to the testing matrix

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

**Review by nemanja-boric-sociomantic — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -19,9 +19,7 @@ env:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
     matrix:
-        - DMD=1.080.0 DIST=xenial F=production AFTER_SCRIPT=1
-        - DMD=1.080.0 DIST=xenial F=devel
-        - DMD=1.081.0 DIST=xenial F=production
+        - DMD=1.081.0 DIST=xenial F=production AFTER_SCRIPT=1
```
Comment (leandro-lucarella-sociomantic) [inline .travis.yml:22]: Should we already?

**Review by leandro-lucarella-sociomantic — COMMENTED** [review]
Travis says no.

**Comment by leandro-lucarella-sociomantic** [comment]
Also, not very important, but it might be a good idea to be consistent about the prefix to the commit message, I used mainly `travis: ` and I see you use `CI: `. I don't really mind which to use, and I guess we'll forget anyway, but it might worth a try to have some sort of conventions :)

File: .travis.yml (hunk 1 above)
Comment (mihails-strasuns-sociomantic) [reply .travis.yml:22]: Definitely, it is both tested and can be used as 100% drop-in replacement.

**Comment by mihails-strasuns-sociomantic** [comment]
Restarting travis after fixing D1 buid in other PR.

**Comment by mihails-strasuns-sociomantic** [comment]
> but it might worth a try to have some sort of conventions :)

`CI:` is shorter allowing slightly longer title before you hit the limit :P

**Comment by mihails-strasuns-sociomantic** [comment]
Green

**Pull request merged**